network/src/validator_network/network_builder.rs

```diff
@@ -413,8 +402,6 @@ impl NetworkBuilder {
         let conn_mgr_reqs_tx = self
             .conn_mgr_reqs_tx()
             .expect("ConnectivityManager not enabled");
-        let (signing_private_key, _signing_public_key) =
-            self.signing_keypair.take().expect("Signing keys not set");
```

Comment:
Would it be appropriate to at least comment on the future of `add_gossip_discovery(&mut self)` above? maybe a [deprecation](https://doc.rust-lang.org/edition-guide/rust-2018/the-compiler/an-attribute-for-deprecation.html)?